packages/client/modules/userDashboard/components/OrgBilling/OrgBillingCreditCardInfo.tsx

```diff
@@ -99,6 +101,7 @@ const OrgBillingCreditCardInfo = (props: Props) => {
           <CreditCardModal
             activeUserCount={activeUserCount}
             orgId={orgId}
+            onUpgrade={onUpgrade}
             actionType={'update'}
```

Comment:
since actionType is `update` onUpgrade doesn't need to be supplied here.
removing it here means this whole component doesn't need to know about the invoicelistrefetch